packages/atomic/src/components/common/no-items/magnifying-glass.ts

```diff
@@ -0,0 +1,13 @@
+import {FunctionalComponent} from '@/src/utils/functional-component-utils';
+import {html} from 'lit';
+import MagnifyingGlassIcon from '../../../images/magnifying-glass.svg';
+
+export const renderMagnifyingGlass: FunctionalComponent<{}> = () => {
```

Comment:
Nitpick.

Perhaps we could include a default type for the `FunctionComponent` generic interface. This way we wouldn't have to include `<{}>` when a functional component has no props.

E.g., 

```typescript
// functional-component-utils.ts

export interface FunctionalComponent<T = Record<string, never>> {
  ({props}: {props: T}): TemplateResult | typeof nothing;
}

// ...
```
```